mlrun/model_monitoring/applications/base.py

```diff
@@ -777,16 +785,18 @@ def evaluate(
         :param write_output:      Whether to write the results and metrics to the time-series DB. Can be ``True`` only
                                   if ``endpoints`` are passed.
                                   Note: the model monitoring infrastructure must be up for the writing to work.
-        :param allow_unordered_data: Relevant only when writing outputs to the database. When ``False``, and the
-                                     requested ``start`` time precedes the ``end`` time of a previous run that also
-                                     wrote to the database - an error is raised.
-                                     If ``True``, when the previously described situation occurs, the relevant time
-                                     window is cut so that it starts at the earliest possible time after ``start``.
+        :param fail_on_overlap:   Relevant only when ``write_output=True``. When ``True``, and the
+                                  requested ``start`` time precedes the ``end`` time of a previous run that also
+                                  wrote to the database - an error is raised.
+                                  If ``False``, when the previously described situation occurs, the relevant time
+                                  window is cut so that it starts at the earliest possible time after ``start``.
         :param stream_profile:    The stream datastore profile. It should be provided only when running locally and
                                   writing the outputs to the database (i.e., when both ``run_local`` and
                                   ``write_output`` are set to ``True``).
                                   For more details on configuring the stream profile, see
                                   :py:meth:`~mlrun.projects.MlrunProject.set_model_monitoring_credentials`.
+        :param application_name:  The application name to use for the results. If not set, the class name is used.
+                                  A ``"-batch"`` suffix is guaranteed to be added if not already there.
```

Comment:
Why we don't use the function_name as we use in the realtime case?

Reply:
Done and removed `application_name`